it/uk/gov/hmrc/soletraderidentificationfrontend/utils/WiremockHelper.scala

```diff
@@ -108,6 +114,34 @@ object WiremockHelper extends Eventually with IntegrationPatience {
     verifyPost("/write/audit/merged")
   }
 
+  def verifyAuditTypeFor(auditTypeToBeFound: String): Unit = {
+    def extractRequestBody(request: LoggedRequest): JsValue = Try(Json.parse(request.getBodyAsString)) match {
+      case Failure(_) => throw new IllegalStateException(s"Audit should receive json request but I did not. Details $request")
```

Comment:
I think "Audit should receive json request but I did not." should be "Audit should receive json request but it did not."

Reply:
going to fix it